From 91e2e3bd4a881755d84fe0f96debb86573c5b238 Mon Sep 17 00:00:00 2001
From: Joachim Bauch <bauch@struktur.de>
Date: Fri, 25 Sep 2020 12:02:08 +0200
Subject: [PATCH] Import "en265.h" to get correct name mangling of exported
 functions.

Without the header, "C++" name mangling will be used instead of the
required "C" names.
---
 libde265/image.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libde265/image.cc b/libde265/image.cc
index 543e74b1..07e4a22d 100644
--- a/libde265/image.cc
+++ b/libde265/image.cc
@@ -20,6 +20,7 @@
 
 #include "image.h"
 #include "decctx.h"
+#include "en265.h"
 
 #include <stdlib.h>
 #include <string.h>
